packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -272,28 +283,110 @@ function MapAnalysisImpl(props: ImplProps) {
     [markerConfigurations, setMarkerConfigurations]
   );
 
+  const filtersIncludingViewport = useMemo(() => {
```

Comment:
**NOTE**: No logic/code has changed for `filtersIncludingViewport` but I had to move its initialization above `allVisibleCategoricalValues`.